Sara/index.js

```diff
@@ -0,0 +1,8 @@
+// with Math.random() ==> get random item from colors array, if the item is number set it to '' 
+const colors = ["red", 9999, "green", "blue", 777, "yellow", "black", "purple", "brown"] ; // 
+console.log('length of colors ==>', colors.length) ; // 9
+let randomItem = colors[Math.floor(Math.random() * 9)] ; 
```

Comment:
You still get number in console, in line  4 let randomItem , get rid of 9 and add colors.length instead
 and after that make an if asatement like this to replace numbers to string
   const pickColor = isNaN(randomItem) === true ? randomItem : "";
  console.log(pickColor);